courses/utils.py

```diff
@@ -173,3 +173,20 @@ def get_unenrollable_courses(queryset):
         .filter(courseruns__id__in=courseruns_qs.values_list("id", flat=True))
         .distinct()
     )
+
+
+def get_archived_courseruns(queryset):
+    """
+    Returns course runs that are archived. This is defined as:
```

Comment:
An archived course is a course that is enrollable, end date has passed.

Reply:
It was meant to say end. It matches the logic in frontend/public/src/lib/courseApi.js this way. If we think it should differ, it needs to be changed in both.